internal/mempool/mempool.go

```diff
@@ -2125,13 +2156,17 @@ func (mp *TxPool) VerboseTxDescs() []*VerboseTxDesc {
 // concurrent access as required by the interface contract.
 func (mp *TxPool) MiningDescs() []*mining.TxDesc {
 	mp.mtx.RLock()
+	defer mp.mtx.RUnlock()
```

Comment:
We prefer to avoid the extra overhead of defer in trivial cases:

```Go
mp.mtx.RLock()
descs := mp.miningDescs()
mp.mtx.RUnlock()
return descs
```

Reply:
fixed